lookout/style/format/__main__.py

```diff
@@ -4,6 +4,7 @@
 from typing import Any
 
 from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
+from lookout.core.slogging import setup
```

Comment:
`as setup_slogging` or just `import slogging` - otherwise `setup` is a very common name